modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -15,6 +15,136 @@
+
+[discrete]
+=== Show Audio and Image EXIF Metadata
+
+Introducing new metadata details, Infinite Scale now features an `EXIF`` panel that displays image metadata whenever available. These metada are shown from image `EXIF` data:
+
+* Dimensions
+* Camera Make
+* Camera Model
+* Focal Length
+* f-number (aperture)
+* Exposure Time
+* ISO
+* Orientation
+* Taken Date and Time
+* Location
+
+Additionally, an Audio Info panel showcases audio metadata: 
+
+* Album
+* Artist
+* Album Artist
+* Genre
+* Title
+* Duration
+* Track
+* Disc
+* Year
+
+[discrete]
+=== Other Notable Changes
+
+* Change - Define maximum input image dimensions and size when generating previews: #9360
+* Enhancement - Limit concurrent thumbnail requests. The number of concurrent requests to the thumbnail service can be limited now to have more control over the consumed system resources. #9199 #9199
+* Enhancement - Change Cors default settings. We have changed the default CORS settings to set Access-Control-Allow-Origin to the OCIS_URL if not explicitely set and Access-Control-Allow-Credentials to false if not explicitely set. #8514 #8518
+* Enhancement - Custom WEB App Loading. We've added a new feature which allows the administrator of the environment to provide custom web applications to the users. This feature is useful for organizations that have specific web applications that they want to provide to their users. The users will then be able to access these custom web applications from the web ui. For a detailed description of the feature, please read the WEB service README.md file. #8392 #8523
+* Enhancement - Make server side space templates production ready. Fixes several smaller bugs and adds some improvements to space templates, introduced with #8558 #8723
+* Enhancement - Allow to resolve public shares without the ocs tokeninfo endpoint. Instead of querying the /v1.php/apps/files_sharing/api/v1/tokeninfo/ endpoint, a client can now resolve public and internal links by sending a PROPFIND request to /dav/public-files/{sharetoken} authenticated clients accessing an internal link are redirected to the "real" resource (`/dav/spaces/{target-resource-id} authenticated clients are able to resolve public links like before. For password protected links they need to supply the password even if they have access to the underlying resource by other means. unauthenticated clients accessing an internal link get a 401 returned with WWW-Authenticate set to Bearer (so that the client knows that it need to get a token via the IDP login page. unauthenticated clients accessing a password protected link get a 401 returned with an error message to indicate the requirement for needing the link's password. #8858 #8926 cs3org/reva#4653
```

Comment:
```suggestion
* Enhancement - Allow public shares to be resolved without the ocs tokeninfo endpoint. Instead of querying the /v1.php/apps/files_sharing/api/v1/tokeninfo/ endpoint, a client can now resolve public and internal links by sending a PROPFIND request to /dav/public-files/{sharetoken} Authenticated clients accessing an internal link are redirected to the "real" resource (`/dav/spaces/{target-resource-id} Authenticated clients are able to resolve public links like before. For password protected links they need to supply the password even if they have access to the underlying resource by other means. Unauthenticated clients accessing an internal link get a 401 returned with WWW-Authenticate set to Bearer (so that the client knows that it needs to get a token via the IDP login page. Unauthenticated clients accessing a password protected link get a 401 returned with an error message to indicate the requirement for providing the link's password. #8858 #8926 cs3org/reva#4653
```